packages/client/src/declarations/selectors.ts

```diff
@@ -29,11 +30,16 @@ export const getInitialDeclarationsLoaded = (
 
 export const selectDeclaration =
   <T extends IDeclaration | undefined>(declarationId: string) =>
-  (store: IStoreState) =>
-    getKey(store, 'declarations').find(({ id }) => declarationId === id) as T
+  (store: IStoreState) => {
+    const bar = getKey(store, 'declarations').find(
+      ({ id }) => declarationId === id
+    ) as T
+    return bar
+  }
 
 export const useDeclaration = <T extends IDeclaration | undefined>(
   declarationId: string
 ) => {
-  return useSelector(selectDeclaration<T>(declarationId))
+  const foo = useSelector(selectDeclaration<T>(declarationId))
+  return foo
```

Comment:
I guess these changes you are meaning to revert still?

Reply:
Yeah!

Reply:
yeah